bears/js/ESLintBear.py

```diff
@@ -31,15 +31,23 @@ class ESLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         eslint_config: str=""):
+                         eslint_config: str=''):
         """
         :param eslint_config: The location of the .eslintrc config file.
         """
         args = '--no-ignore', '--no-color', '-f=json', '--stdin'
+        if not eslint_config:
+            args += ('--config', config_file)
+
         if eslint_config:
             args += ('--config', eslint_config)
```

Comment:
@sils1297 are you sure there won't be an issue because of two `--config` flags? They'll be appended internall?